index.bs

```diff
@@ -1231,7 +1231,9 @@ dictionary SensorErrorEventInit : EventInit {
 
 ### SensorErrorEvent.error ### {#sensor-error-event-error}
 
-Gets the {{DOMException}} object passed to {{SensorErrorEventInit}}.
+The {{SensorErrorEvent/error!!attribute}} getter steps are to return the value it was initialized to.
+
+It represents the {{DOMException}} object passed to {{SensorErrorEventInit}}.
```

Comment:
As a follow-up we should really properly specify the constructor algorithm and internal slots for SensorErrorEvent instead of this hand-wavy description.

Reply:
Agreed, I opened an issue for that: https://github.com/w3c/sensors/issues/426